src/Workshop/EmailNotifier.php

```diff
@@ -40,13 +40,17 @@ public function __construct(SendGrid $sendGrid, string $phpSchoolEmail)
 
     public function approved(Workshop $workshop)
     {
+        return;
```

Comment:
Apparently our send grid account has been removed so no point fixing up this code yet, since the API has changed massively.